r-pkg/R/get_fields.R

```diff
@@ -243,52 +242,13 @@ get_fields <- function(es_host
         # there are no aliases in this Elasticsearch cluster
         return(invisible(NULL))
     } else {
-        major_version <- .get_es_version(es_host)
-        process_alias <- switch(
-            major_version
-            , "1" = .process_legacy_alias
-            , "2" = .process_legacy_alias
-            , "5" = .process_new_alias
-            , "6" = .process_new_alias
-            , .process_new_alias
```

Comment:
Turns out we didn't need any of this stuff!!!

I realized through this exercise that the difference across Elasticsearch versions was like this:

* we weren't passing the `Accept` header in this request to `GET /_cat/aliases`
* on ES 1.x and 2.x, `GET /_cat/aliases` responded with fixed-width text by default
* from ES 3.x onwards, it responded with JSON data by default
* across ALL versions, it responds with JSON data if you pass `Accept: application/json`

So just passing that header, this now always returns JSON and no more need for these different processing functions 😁 😁 😁

Reply:
Amazing!